code/modules/surgery/robotics.dm

```diff
@@ -338,7 +338,7 @@
 		if(!istype(M))
 			return -1
 
-		if(!M.brainmob || !M.brainmob.client || !M.brainmob.ckey || M.brainmob.stat >= DEAD)
+		if(!M.brainmob || !M.brainmob.client || !M.brainmob.ckey || M.brainmob.stat == DEAD)
```

Comment:
This is weird. Why -- what the heck is a brainmob? hold on. brb.

Reply:
Ooooh~ M is an MMI and Brainmob is the brain that's in it. This is probably why Baldbotics will throw away good brains on Paradise. They stick it into an MMI, nope, doesn't work. This statement should be clarified.